KarhooUISDK/CommonUI/UITexts.swift

```diff
@@ -14,462 +14,446 @@ public enum UITexts {
+        public static let prebookConfirmation = "kh_uisdk_prebookConfirmation".localized
+        public static let noAvailabilityHeader = "kh_uisdk_quotes_no_availability".localized
+        public static let noAvailabilityBody = "kh_uisdk_noAvailabilityBody".localized
+        public static let noAvailabilityLink = "kh_uisdk_noAvailabilityLink".localized
+        public static let guestCheckoutPassengerDetailsTitle = "kh_uisdk_passenger_details".localized
+        public static let guestCheckoutPaymentDetailsTitle = "kh_uisdk_guestCheckoutPaymentDetailsTitle".localized
+        public static let guestCheckoutFlightNumberPlaceholder = "kh_uisdk_flight_number".localized
+        public static let estimatedInfoBox = "kh_uisdk_price_info_text_estimated".localized
+        public static let meteredInfoBox = "kh_uisdk_price_info_text_metered".localized
+        public static let fixedInfoBox = "kh_uisdk_price_info_text_fixed".localized
+        public static let learnMore = "kh_uisdk_quote_learn_more".localized
+        public static let passenger = "kh_uisdk_booking_checkout_passenger".localized
+        public static let maximumPassengers = "kh_uisdk_passengers_max_ios".localized
+        public static let maximumLuggages = "kh_uisdk_baggage_max_ios".localized
+        public static let gpsTracking = "kh_uisdk_filter_gps_tracking".localized
+        public static let trainTracking = "kh_uisdk_filter_train_tracking".localized
+        public static let flightTracking = "kh_uisdk_filter_fight_tracking".localized
+        public static let quoteExpiredTitle = "kh_uisdk_offer_expired".localized
+        public static let quoteExpiredMessage = "kh_uisdk_offer_expired_text".localized
+        public static let legalNotice = "kh_uisdk_legal_notice_label".localized
+        public static let legalNoticeLink = "kh_uisdk_legal_notice_link".localized
+        public static let legalNoticeText = "kh_uisdk_legal_notice_text".localized
+        public static let legalNoticeTitle = "kh_uisdk_legal_notice_title".localized
+        public static let noLocationPermissionTitle = "kh_uisdk_noLocationPermissionTitle".localized
+        public static let noLocationPermissionMessage = "kh_uisdk_noLocationPermissionMessage".localized
+        public static let noLocationPermissionConfirm = "kh_uisdk_settings".localized
     }
 
     public enum Availability {
-        public static let allCategory = "Text.Availability.AllCategory".localized
-        public static let noQuotesInSelectedCategory = "Text.Availability.NoQuotesInSelectedCategory".localized
-        public static let noQuotesForSelectedParameters = "Text.Availability.NoQuotesForSelectedParameters".localized
+        public static let allCategory = "kh_uisdk_filter_all".localized
+        public static let noQuotesInSelectedCategory = "kh_uisdk_noQuotesInSelectedCategory".localized
+        public static let noQuotesForSelectedParameters = "kh_uisdk_K3002".localized
```

Comment:
Looks like a wrong key was mapped. Please double check and ignore the comment if the mapping is correct